pkg/relayer/session/session_retry_test.go

```diff
@@ -0,0 +1,402 @@
+package session_test
```

Comment:
Note to reviewer: Testing the `RelayerSessionsManager` retry integration is not finished yet.